step.yml

```diff
@@ -0,0 +1,46 @@
+title: Flank
```

Comment:
What about: Flank test?

Reply:
The step name is already reviewed and discussed by the proposal. Flank test for me means a new type of test and not the test runner what Flank really is.